<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>454</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Sscom For Qt by zhuyu4839@gmail.com</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="txtSendRecv">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayoutLeftCenter">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayoutLeftCenterHead">
            <item row="0" column="8">
             <widget class="QPushButton" name="btnShowHidden">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Show</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="btnStopSendFile">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="btnClearSend">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Clear Send</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnClearRecv">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Clear Recv</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="txtFilePath">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="QPushButton" name="btnSaveParam">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Save Param</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btnOpenFile">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="btnSendFile">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Send File</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QCheckBox" name="cBoxTop">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Top</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayoutLeftCenterTail">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayoutLeftCenterTailLeft">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelMode">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Mode:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="cbBoxMode">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="btnConnect">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Connect</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QFrame" name="frameNetwork">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="0" column="0">
                    <widget class="QLabel" name="labelTarget">
                     <property name="maximumSize">
                      <size>
                       <width>80</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Target:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QPushButton" name="btnDisconnect">
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Disconnect</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="txtTarget">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QComboBox" name="cbBoxHost">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="editable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QLineEdit" name="txtHostPort">
                     <property name="maximumSize">
                      <size>
                       <width>60</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="maxLength">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLineEdit" name="txtTargetPort">
                     <property name="maximumSize">
                      <size>
                       <width>60</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="maxLength">
                      <number>10</number>
                     </property>
                     <property name="placeholderText">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="labelHost">
                     <property name="maximumSize">
                      <size>
                       <width>80</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Host:</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="gridLayoutLeftCenterTailRight">
              <item row="0" column="0">
               <widget class="QCheckBox" name="cBoxShowHex">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Show Hex</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4" colspan="3">
               <widget class="QCheckBox" name="cBoxSendHex">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Send Hex</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QCheckBox" name="cBoxIntervalSend">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Interval Send</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string notr="true">-</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QLineEdit" name="txtEnd">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>End</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2" colspan="2">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelTimeout">
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>TimeOut:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="txtTimeout">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="maxLength">
                   <number>10</number>
                  </property>
                  <property name="placeholderText">
                   <string notr="true">ms</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="cBoxAutoCR">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Auto CR</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QPushButton" name="btnSaveData">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>SaveData</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QCheckBox" name="cBoxSave2File">
                <property name="maximumSize">
                 <size>
                  <width>190</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Save Recv to File</string>
                </property>
               </widget>
              </item>
              <item row="1" column="7">
               <widget class="QLabel" name="label_1">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Byte Verify</string>
                </property>
               </widget>
              </item>
              <item row="0" column="8">
               <widget class="QLineEdit" name="txtInvervalSend">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="placeholderText">
                 <string notr="true">ms</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2" colspan="2">
               <widget class="QCheckBox" name="cBoxAutoLF">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Auto LF</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QCheckBox" name="cBoxShowTimestamp">
                <property name="maximumSize">
                 <size>
                  <width>180</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Show TimeStamp</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="8">
               <widget class="QComboBox" name="cbBoxVerify">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>None</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLineEdit" name="txtStart">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="placeholderText">
                 <string notr="true">1</string>
                </property>
               </widget>
              </item>
              <item row="2" column="7">
               <widget class="QPushButton" name="btnSend">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="txtSend">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>135</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClearSend</sender>
   <signal>clicked()</signal>
   <receiver>txtSend</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>972</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearRecv</sender>
   <signal>clicked()</signal>
   <receiver>txtSendRecv</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
